easybuild/easyconfigs/k/Kraken/Kraken-1.1-foss-2018b-Perl-5.28.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PackedBinary'
+
+name = 'Kraken'
+version = '1.1'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://ccb.jhu.edu/software/%(namelower)s/'
+description = """Kraken is a system for assigning taxonomic labels to short DNA sequences,
+ usually obtained through metagenomic studies. Previous attempts by other
+ bioinformatics software to accomplish this task have often used sequence
+ alignment or machine learning techniques that were quite slow, leading to
+ the development of less sensitive but much faster abundance estimation
+ programs. Kraken aims to achieve high sensitivity and high speed by
+ utilizing exact alignments of k-mers and a novel classification algorithm."""
+
+# Part is compiled with CXX, the rest is in Perl
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'openmp': True}
+
+github_account = 'DerrickWood'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_CXX-CXXFLAGS.patch']
+checksums = [
+    'a4ac74c54c10920f431741c80d8a172670be12c3b352912000030fb5ea4c87a7',  # v1.1.tar.gz
+    '84c017d6a80ccaac1e23561c83cac90bcc3d62baa8d31657a7dafbc2c3f29726',  # Kraken-1.1_CXX-CXXFLAGS.patch
+]
+
+dependencies = [
+    ('Perl', '5.28.0'),
+    ('Jellyfish', '1.1.12'),
+    ('wget', '1.20.1', '-GCCcore-7.3.0', True),
```

Comment:
@SethosII This should just be `('wget', '1.20.1')`?

Reply:
@boegel I changed this.